### PR TITLE
Task_52982 - Implement Homepage

### PR DESCRIPTION
**Pull request by TirlaP**

### CODE REVIEW

File: src/UI/dropdown/Dropdown.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+import FormHelperText from "@mui/material/FormHelperText";
+import FormControl from "@mui/material/FormControl";
+import Select from "@mui/material/Select";
+
+import styled from "@emotion/styled";
+
+const CustomSelect = styled(Select)(() => ({
+	"&": {
+		margin: "0px",
+		width: "178px",
```
Comment (petricafinaga) [inline src/UI/dropdown/Dropdown.js:13]: I think you should get rid of this style-components and just use normal css

File: src/components/carousel/Carousel.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+};
+
+export default Carousel;
+
+{
+	/* 
+		Cum fac functionalitatea pentru ultimul card din Slider?
+
+		1. Adaug la final un card gol
+		- doar ca acel card o sa ia ceva imagine/ titlu/ locatie/ pret, lucruri pe care nu trebuie sa le afisez
```
Comment (petricafinaga) [inline src/components/carousel/Carousel.js:86]: If you don't need this comment anymore you can remove it

File: src/components/carousel/CarouselStyles.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+	color: "#0356E8",
+	width: "20px",
+	height: "20px",
+	borderRadius: "50%",
+	marginRight: "-30px",
+	zIndex: "200",
+	".MuiSvgIcon-root": {},
+};
+
+export default css.global``;
```
Comment (petricafinaga) [inline src/components/carousel/CarouselStyles.js:25]: If you are not using any styled-jsx here you can remove this export and also the import

File: src/components/common/card/favoriteButton/FavoriteBtn.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+		<div className="circle-fav">
+			<Avatar sx={favBtn}>
+				<FavoriteBorderIcon />
+			</Avatar>
+			{/* <FavoriteBorderIcon
+				sx={{
+					position: "absolute",
+					left: "8.33%",
+					right: "8.33%",
+					top: "11.21%",
```
Comment (petricafinaga) [inline src/components/common/card/favoriteButton/FavoriteBtn.js:27]: If you won't need this piece of code in the future, you can remove it :)

File: src/pages/homepage/HomePage.js [hunk 5]
```diff
@@ -1,9 +1,64 @@
...
+		if (card.category === "Apartments") {
+			return card;
+		}
+	});
+
+	const sortedDate = [...CardsData].sort(
+		(objA, objB) => Number(new Date(objB.date)) - Number(new Date(objA.date))
+	);
+	// const jsonDate = new Date(objB.date);
+	// console.log(jsonDate.getDay());
```
Comment (petricafinaga) [inline src/pages/homepage/HomePage.js:38]: This commented code, and also the logs can be removed

**Review by petricafinaga — CHANGES_REQUESTED** [review]

**Review by petricafinaga — APPROVED** [review]

**Review by dianamiraute — APPROVED** [review]

**Pull request merged**